src/core/metrics.h

```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2018-2020, NVIDIA CORPORATION. All rights reserved.
+// Copyright 2018-2021 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
```

Comment:
Here and in other places keep (c) after Copyright. Also the convention is:
"Copyright (c) 2020-21 ..."